.travis.yml

```diff
@@ -53,19 +53,24 @@ before_script:
 script:
   - make test-dirty test
 
+jobs:
+  include:
+    - stage: deploy
+      env:
+        - OPERATOR_SDK_VERSION=v0.8.1 KUBE_VERSION=v1.15.10 MINIKUBE_VERSION=v1.6.2 SHFMT_VERSION=v3.0.1
```

Comment:
@seanmalloy why we're not use vars from the beginning of this yml? Smth like `OPERATOR_SDK_VERSION=${OPERATOR_SDK_VERSION}`

Reply:
Without setting the environment variables the `deploy` runs multiple times. Once for each matrix dimension(item?). So we end end running the `deploy` twice, once for each `KUBE_VERSION` in the matrix.

It might be possible to use `jobs.include` in make this cleaner. But I think we need to do something explicit in the YAML to only run the `deploy` a single time.

Reply:
I'd prefer to keep this as is because I know it is working.

Reply:
Let me know what you think.

Reply:
I re-read the doc  ["Build Stages and Build Matrix Expansion"](https://docs.travis-ci.com/user/build-stages/#build-stages-and-build-matrix-expansion). The implicit build matrix include feature does not work in our case. This only works when specifying the language version, which is in our case Go(see line 9).

We have to explicitly include `env` for the deploy jobs because we also have environment variables in our matrix too(see lines 14 to 19).

Summary: I believe we have to do it the way I have proposed in this PR, or the deploy jobs will run once for each `KUBE_VERSION` in the matrix.